class/defaults.yml

```diff
@@ -116,3 +116,7 @@ parameters:
         PROJECT_DESCRIPTION: {}
         PROJECT_ADMIN_USER: {}
         PROJECT_REQUESTING_USER: {}
+
+    runOnceActiveDeadlineSeconds:
+      defaultActiveDeadlineSeconds: 1800
+      overrideAnnotationKey: appuio.io/activeDeadlineSecondsOverride
```

Comment:
Do we have a style guide for label naming (`activeDeadlineSecondsOverride` vs `active-deadline-seconds-override`)? While reviewing github.com/appuio/appuio-io-docs/pull/63, I felt that using `-` is more appropriate. This is also what Red Hat is doing.

Reply:
Actually, I copied this style from the OCP3 `openshift.io` annotation, but no hard feelings either way.